eng/SourceBuild.props

```
@@ -0,0 +1,23 @@
+<Project>
+
+  <PropertyGroup>
+    <GitHubRepositoryName>nuget-client</GitHubRepositoryName>
```

Shoud it be  NuGet.Client as the repo name?

```suggestion
    <GitHubRepositoryName>NuGet.Client</GitHubRepositoryName>
```

https://github.com/dotnet/source-build/blob/master/Documentation/planning/arcade-powered-source-build/onboarding/local-onboarding.md#engsourcebuildprops

---

Yes, thanks.